### ISSUE TITLE
Cannot find required library with namespace

### ISSUE THREAD

**Issue by oetiker** [qooxdoo/qooxdoo-compiler#341]
it seems that requirement for contribs listed in Manifest.json do not work

```console
$ qx contrib install cboulanger/eventrecorder
$ qx compile
(node:79400) ExperimentalWarning: The fs.promises API is experimental
qxapp: Cannot find required library with namespace cboulanger/eventrecorder
```
Manifest.json reads
```
"requires": {
    "qooxdoo-sdk": "^6.0.0-alpha",
    "qooxdoo-compiler": "^0.3.0-alpha",
    "cboulanger/eventrecorder": "^0.8.0"
  }
````
Running on: qooxdoo-compiler v0.3.0-alpha.20190326-1007

Note that the generated application still works ... it seems as if the requires config in Manifest.json has only cosmetic purposes for now.

This is not specific to eventrecorder ... I just used it as an example.

**Comment by hkollmann** [comment]
Could you try version 0.3.0-alpha.20190326-1006? I fixed this - could be an overlapped merge problem...

**Comment by oetiker** [comment]
indeed 1006 does work :)

**Comment by cboulanger** [comment]
Ok, this is what I see with newest qxcompiler from nom: 
```
qx create foo -I
cd foo/
qx contrib install cboulanger/eventrecorder
ergobyte/qookery/qookery is not an contrib!
```
after adding
`"include": ["cboulanger.eventrecorder.UiController", "cboulanger.eventrecorder.ObjectIdGenerator"]`
 to the an application entry in `compile.json`, I get

```
qx serve
>>> Making the applications...
Cannot find class file cboulanger.eventrecorder.ObjectIdGenerator
Cannot find class file cboulanger.eventrecorder.UiController
```
Looking at the files it seems `contrib.json` has not been generated.

**Comment by cboulanger** [comment]
and the "ergobyte/qookery/qookery is not an contrib!" message is also strange (and there is a spelling mistake, it should be "not a contrib".

**Comment by cboulanger** [comment]
Ok, of course. This is about inter-contrib dependencies. There is a problem that the compile.json file is not created when one dependency of a contrib isn't found.

**Comment by cboulanger** [comment]
This should fix it: #342

**Comment by hkollmann** [comment]
https://github.com/qooxdoo/qooxdoo-compiler/pull/340 fixes this issue. Because of the 2 merge i the same minute from John the second build has not recognized the previous one. 
So a dummy commit should fix the problem.

**Comment by hkollmann** [comment]
@oetiker tried the version 1006 wich is the previous build and that version works.

**Comment by hkollmann** [comment]
So we have 2 problems. One with qx compile fixed with #340 and one with install fixed with #342,

**Comment by oetiker** [comment]
just tested with latest release looking good now
